src/serlio/modifiers/MayaCallbacks.h

```diff
@@ -43,9 +45,12 @@ class MayaCallbacks : public IMayaCallbacks {
 		LOG_ERR << "GENERATE ERROR: " << message;
 		return prt::STATUS_OK;
 	}
-	prt::Status assetError(size_t /*isIndex*/, prt::CGAErrorLevel /*level*/, const wchar_t* /*key*/,
+	prt::Status assetError(size_t /*isIndex*/, prt::CGAErrorLevel level, const wchar_t* /*key*/,
 	                       const wchar_t* /*uri*/, const wchar_t* message) override {
 		LOG_ERR << "ASSET ERROR: " << message;
+		if (message != nullptr && wcsstr(message, L"CGAC version")) {
```

Comment:
please pull the detection of a "CGAC mismatch" event into a separate function

Reply:
fixed